docs/src/pages/company/careers/product-designer.md

```diff
@@ -1,92 +1,74 @@
 
-So far we have relied a lot on the Material Design guidelines to implement amazing-looking components. However, we have reached the point where this is no longer sufficient.
+- [MUI Core:](/core)
+  - [MUI Base:](/customization/unstyled-components/) the unstyled version of the components.
+  - [MUI System:](/system/basics/) our most recent take on styling for React components.
+  - [MUI Joy (working title)](https://github.com/mui-org/material-ui/discussions/29024): a second design system, alternative to Material Design.
+- [MUI X:](/x) as mentioned already, a growing set of advanced components. Today, the flagship is the [Data Grid](/components/data-grid/), a game-changing component for dealing with lots of data, which integrates perfectly with MUI Core.
+- MUI Studio: a very recent endeavor aimed at exploring how can our users visually create apps 10x faster with the power of low-code and the flexibility of pro-code.
 
-We are aiming to make high-end design accessible to the many, empowering developers that lack access to a designer or dislike writing CSS.
-We are looking for someone that can contribute to the following outcomes:
-
-- Make the React component library the most visually appealing in the market.
-- Strengthen the existing design team to support the growing number of developers working at MUI.
-- Improve the product direction.
-
-### Why this is interesting
-
-The quality of design execution is [the leading reason why](/blog/2020-developer-survey-results/#6-what-are-your-key-criteria-when-choosing-a-ui-library) people pick MUI.
-Providing functional components isn't enough, far from it.
-We will empower developers & designers to build beautiful UIs 😍.
-
-Our solution empowers React developers to build awesome applications – hundreds of thousands of developers use MUI every month.
+We also know, especially due to [our yearly run developer surveys](/blog/2020-developer-survey-results/), that design quality plays a huge part when developers are considering available component library options. Therefore, we need to grow the design organization to help us push these initiatives further.
 
 ## About the role
 
-### What you'll do on a day-to-day basis
+Some criterias for applying to this role:
 
-Depending on the day, you'll:
+- Location: fully remote (preference for UTC-6 to UTC +3).
+- Type of work: full-time (as a contractor or employee, [depending on the case](https://www.notion.so/Hiring-FAQ-64763b756ae44c37b47b081f98915501)).
+- Level: [2 or above](https://docs.google.com/spreadsheets/d/18VvbVCNBRWu-QYPncBl5IWTXTOpD68UQsLsymBvfm88/edit#gid=0).
```

Comment:
Are we sure about level 2 as the minimum required for this role? Level 2 is right after the junior level. This got me thinking:

- For new departments, shouldn't aim to hire as high level people as we can find to begin with? Priority to execution, and then, as we grow, start to onboard lower levels? This is the strategy that we did for all the hires so far. The more entry levels only got in after we got ~4 people in the team.
- The designer role for the studio is key, enough to give it more priority than a developer. Level 2 doesn't seem to reflect it.
- I had a look at the level grid, to see what I expect that we need for this role. The only dimensions that I have found relevant are execution and strategy, I think 4 would make more sense on them.

Maybe to keep it open ended for the applications?
```suggestion
- Level: [3 or above](https://docs.google.com/spreadsheets/d/18VvbVCNBRWu-QYPncBl5IWTXTOpD68UQsLsymBvfm88/edit#gid=0).
```

Reply:
I changed to Level 2 because you hinted on Notion that starting off Level 3 wasn't a good idea. If we're going to change it, I think it should be for Level 3 (senior). This is a role that requires managing work from two different teams and having the autonomy and enough sense of prioritization to do so, that's why I think aiming for seniors makes more sense.

Reply:
From what I recall in https://www.notion.so/mui-org/Designer-job-ad-page-bd6818d4dd1648ac8ce3180b59eb27e0, we were discussing the dimensions that the compensation should factor in. We had only the location, and I was pushing for adding the profile too. 
I had the feeling that the root cause of this was either we should be hiring for a specific level or between a minimum level and upward.

Are you ok with "Level 3 or above" or would you have it another way? In any case, we can adjust based on the applications that we receive.

Reply:
I'm good with 3 and above!